### DISCUSSION TITLE
Load content of a file from private area

### DISCUSSION THREAD

**Discussion by Germardies** [community/community#45059] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, everyone,
I'm new here, so have some mercy.
I have an XML file in a private repositories.
Later in the project, this will no longer be under private, but public.
I have these in the private area as long as the project is not online.
My program (Written in Visual Basic) should read and process this file.
It will become a language file. I would like to have them here later so that I can change them at a central location.
But now I have the problem that I need a link to read this file using VB HttpClient().
When I redeem the file's link, I get a 404 error back from the server.
But I can't seem to read it as long as it's in the private sphere.
Is there a way to temporarily read the file without marking it as public?

**Comment by HumbleDeer** [comment]
Yes, with access tokens.
